bedrock/mozorg/templates/mozorg/home/home-a.html

```diff
@@ -0,0 +1,280 @@
+      <ul id="nav-main-menu">
+        <li class="first about-item">
+          <a href="{{ url('mozorg.about') }}" data-link-type="nav" data-link-name="About">{{ _('About') }}</a>
+        </li>
+        <li class="contribute-item">
+          <a href="{{ url('mozorg.contribute') }}" data-link-type="nav" data-link-name="Participate">{{ _('Participate') }}</a>
+        </li>
+        <li class="products-item">
+          <a href="{{ url('firefox.family.index') }}" data-link-type="nav" data-link-name="Firefox">{{ _('Firefox') }}</a>
+        </li>
+      </ul>
+    </nav>
+  {% endblock %}
+
+  {% block site_header_logo %}
+    <h2 class="masthead-logo">
+      <a href="#intro">
+        <img src="{{ static('img/home/new/mozilla-wordmark.svg') }}" alt="Mozilla">
+      </a>
+    </h2>
+  {% endblock %}
+
+  <nav class="masthead-nav-page" id="nav-page">
+    <ul id="nav-page-primary" class="primary">
+      <li id="nav-primary-action"><a href="#action">{{ _('Mozilla In Action') }}</a></li>
+      <li id="nav-primary-innovation"><a href="#innovation">{{ _('Web Innovations') }}</a></li>
+      <li id="nav-primary-firefox"><a href="#firefox">{{ _('Firefox') }}</a></li>
+    </ul>
+
+    <ul id="nav-page-secondary" class="secondary">
+      <li id="nav-secondary-donate"><a href="{{ donate_url('home_test_a') }}">{{ _('Donate') }}</a></li>
+      <li id="nav-secondary-firefox">
+        {{ download_firefox(alt_copy=_('Download Firefox'), button_color='transparent', dom_id='nav-download-firefox') }}
+      </li>
+    </ul>
+  </nav>
+
+  </div>
+</header>
+{% endblock %}
+
+{% block content %}
+<main role="main" class="main">
+  <section class="section section-intro" id="intro">
+    <div id="home-slideshow">
+      <div class="slide" id="slide-one"></div>
+      <div class="slide" id="slide-two"></div>
+      <div class="slide" id="slide-three"></div>
+    </div>
+    <div class="content">
+      <h1 class="intro-tagline">
+        {{ _('We are a non-profit with one mission: defend the freedom of the internet for all') }}
+      </h1>
+    </div>
+  </section>
+
+  <section class="section section-action" id="action">
+    <div class="section-lead">
+      <div class="content">
+        <h2>{{ _('A network, hard at work') }}</h2>
+        <p>{{ _('At Mozilla, we bring together educators, builders, advocates and experts who work fearlessly to protect our individual freedoms on the internet.') }}</p>
+      </div>
+    </div>
+
+    <div id="encryption-video-container" class="moz-video-container">
+      <button class="moz-video-button" type="button" aria-controls="encryption-video">{{ _('Play video') }}</button>
+      <video controls="" id="encryption-video" poster="{{ static('img/home/new/encryption/encryption-video-poster.jpg') }}" preload="metadata">
+        <source id="webm" src="https://d24kjznqej0s8a.cloudfront.net/2016/encryption_campaign/Mozilla_Encrypt_Beat2_landingpage.webm" type="video/webm">
```

Comment:
We appropriated the video straight from [https://advocacy.mozilla.org/encrypt/](https://advocacy.mozilla.org/encrypt/), which I think is fine for the test. (Not sure why they didn't use the regular video CDN...)

If we end up using this prototype as *the* homepage, I think moving the video to the conventional location is a good idea.